library/bz2.po

```diff
@@ -267,6 +349,11 @@ msgid ""
 "to :meth:`decompress`. The returned data should be concatenated with the "
 "output of any previous calls to :meth:`decompress`."
 msgstr ""
+"Décompresse *data* (un :term:`bytes-like object`), renvoyant une donnée non "
+"compressée en tant qu'octet. Certaines de ces *data* peuvent être mis en "
+"interne en tampon, pour un usage lors d'appels ultérieurs par la méthode :"
+"meth:`decompress`. La donnée renvoyée devrait être concaténée avec la sortie "
```

Comment:
```suggestion
"meth:`decompress`. La donnée renvoyée doit être concaténée avec la sortie "
```

Reply:
Là, encore, je ne valide pas . "Should be" signifie "devrait être"... et non pas "doit être"

Reply:
C'est vrai que "should be" se traduit litéralement par "devrait être". Mais on ne traduit pas litéralement. Là (et souvent quand ils utilisent "should" en anglais) si on ne concatène pas, ça ne marche pas, alors ils sont gentils, ils prennent des pincettes, c'est leur style d'expression, mais il *faut* concaténer.

En français on prend pas autant de pincettes : la donnée *doit* être concaténée.